### PR TITLE
INS-2874 we wait NoWait result in loop

### PR DESCRIPTION
**Pull request by aalekseev-ru**
**- What I did**
fix bugs, add loop for noWait call results checking

### CODE REVIEW

File: functest/logicrunner_test.go [hunk 1]
```diff
@@ -381,9 +385,19 @@ func (r *Two) Value() (int, error) {
...
+	testPassed := false
+	for i := 0; i < 25; i++ {
+		resp = callMethod(t, obj, "Value")
+		require.Empty(t, resp.Error)
+		fmt.Println(">>>> resp.ExtractedReply: ", resp.ExtractedReply)
+		if float64(644) == resp.ExtractedReply {
+			testPassed = true
+			break
+		}
+		time.Sleep(1000 * time.Millisecond)
```
Comment (bigbes) [inline functest/logicrunner_test.go]: equal to `time.Second`, but it's better to create "exponential waiting", IMHO

File: functest/logicrunner_test.go [hunk 2]
```diff
@@ -381,9 +385,19 @@ func (r *Two) Value() (int, error) {
...
 	require.Empty(t, resp.Error)
 
-	resp = callMethod(t, obj, "Value")
-	require.Empty(t, resp.Error)
-	require.Equal(t, float64(644), resp.ExtractedReply)
+	testPassed := false
+	for i := 0; i < 25; i++ {
+		resp = callMethod(t, obj, "Value")
+		require.Empty(t, resp.Error)
+		fmt.Println(">>>> resp.ExtractedReply: ", resp.ExtractedReply)
```
Comment (bigbes) [inline functest/logicrunner_test.go]: are you sure you want to `Println` here?

File: functest/logicrunner_test.go (hunk 1 above)
Comment (aalekseev-ru) [reply functest/logicrunner_test.go]: bug situations became only on pulse changes, so i just add time for waiting a few pulses

File: functest/logicrunner_test.go (hunk 2 above)
Comment (aalekseev-ru) [reply functest/logicrunner_test.go]: fixed

File: logicrunner/logicrunner_unit_test.go [hunk 3]
```diff
@@ -1659,3 +1659,40 @@ func (s LRUnsafeGetLedgerPendingRequestTestSuite) TestUnsafeGetLedgerPendingRequ
...
+		}
+
+		proc := initializeExecutionState{
+			LR:  suite.lr,
+			msg: msg,
+		}
+		err := proc.Proceed(suite.ctx)
+		require.NoError(t, err)
+
+		fmt.Println(proc.Result.broker.mutable)
```
Comment (afiskon) [inline logicrunner/logicrunner_unit_test.go]: Please remove this.

**Pull request merged**